packages/python-packages/api-stub-generator/sdk_packaging.toml

```diff
@@ -0,0 +1,6 @@
+[packaging]
+auto_update = false
```

Comment:
Honestly I think you can get rid of `sdk_packaging.toml` as it's only really associated with our management tooling.